x-pack/plugin/core/src/main/resources/ecs-core-mappings.json

```diff
@@ -0,0 +1,318 @@
+{
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "ecs_timestamp": {
+          "mapping": {
+            "ignore_malformed": false,
+            "type": "date"
+          },
+          "match": "@timestamp"
+        }
+      },
+      {
+        "message_match_only_text": {
+          "mapping": {
+            "type": "match_only_text"
+          },
+          "match": "message"
+        }
+      },
+      {
+        "ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "match": "ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "_ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "path_match": "*_ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "wildcard": {
+          "mapping": {
+            "type": "wildcard"
+          },
+          "path_match": [
+            "*.io.text",
+            "registry.data.strings",
+            "*url.path"
+          ]
+        }
+      },
+      {
+        "command_line_to_multifield": {
+          "mapping": {
+            "fields": {
```

Comment:
I don't think we should cover multi fields by default in here. I consider this an extended use case as it consumes extra storage. Any integration using this, should specify it specifically.

Same applies for the fields below.

Reply:
So what would you use for it? This relies on the [ECS mapping](https://www.elastic.co/guide/en/ecs/8.7/ecs-process.html#field-process-command-line) for this field. 
I am afraid that if we start reconsidering each, we may lose the benefit of relying on longstanding accumulation of decisions with considerable milage behind them.

Reply:
Agree, if we call it `ecs` we likely should not deviate unfortunately. I've been discussion with @felixbarny if we should have potentially a third template which is the default that only contains patterns where we could leave these out.